src/core/CCResult.mli

```diff
@@ -185,7 +174,11 @@ val both : ('a, 'err) t  -> ('b, 'err) t -> (('a * 'b), 'err) t
 (** {2 Infix} *)
 
 module Infix : sig
+  val (<$>) : ('a -> 'b) -> ('a, 'err) t -> ('b, 'err) t
```

Comment:
needs a `@since NEXT_RELEASE`  ! :)

Reply:
Just to be clear, am I literally writing the text `NEXT_RELEASE`, or substituting in the version number for the next release (`2.9`?).

Reply:
literally write `@since NEXT_RELEASE` :-)
Have a look in the makefile to see how it's removed. And next version should be 3.0 hopefully.